lib/graphlient/query.rb

```diff
@@ -2,12 +2,24 @@ module Graphlient
   class Query
     attr_accessor :query_str
 
+    ACTIONS = %w(query mutation subscription fragment).freeze
+
     def initialize(&block)
       @indents = 0
```

Comment:
I understand this was here, but I still have a problem with storing `@indents`, I think this should be passed around as a parameter because it's state. If an exception is raised within a block, the next time you call the `to_s` method you're going to get the wrong result.

Reply:
totally see your point, i thought you were talking about `@wrap` which i got rid of completely, will follow up on this in my next already open WIP PR 🤚